CMIOMinimalSample/Stream.mm

```diff
@@ -171,22 +171,16 @@ - (void)fillFrame {
 
     CVPixelBufferRef pixelBuffer = [self createPixelBufferWithTestAnimation];
 
-    // The timing here is quite important. For frames to be delivered correctly and successfully be recorded by apps
-    // like QuickTime Player, we need to be accurate in both our timestamps _and_ have a sensible scale. Using large
-    // timestamps and scales like mach_absolute_time() and NSEC_PER_SEC will work for display, but will error out
-    // when trying to record.
-    //
-    // Instead, we start our presentation times from zero (using the sequence number as a base), and use a scale that's
-    // a multiple of our framerate. This has been observed in parts of AVFoundation and lets us be frame-accurate even
-    // on non-round framerates (i.e., we can use a scale of 2997 for 29,97 fps content if we want to).
-    CMTimeScale scale = FPS * 100;
-    CMTime frameDuration = CMTimeMake(scale / FPS, scale);
-    CMTime pts = CMTimeMake(frameDuration.value * self.sequenceNumber, scale);
+    double hostTime = CACurrentMediaTime();
+    double timescale = 600.0;
```

Comment:
Why is 600 better than 100?

Reply:
Looking at various samples, the timescale value is often set to 600.

Reply:
Those samples are likely running at 60fps, so a 600 result would be fps * 100, just like we're already using.